store/sqldb/sp.go

```diff
@@ -329,9 +332,9 @@ func (s *SpDBImpl) SetStorageParams(params *storagetypes.Params) error {
 	}
 
 	insertParamsRecord := &StorageParamsTable{
-		MaxSegmentSize:          params.GetMaxSegmentSize(),
-		RedundantDataChunkNum:   params.GetRedundantDataChunkNum(),
-		RedundantParityChunkNum: params.GetRedundantParityChunkNum(),
+		MaxSegmentSize:          params.VersionedParams.GetMaxSegmentSize(),
```

Comment:
In fact, params should wraps the VersionedParams in greenfield. The external call method should remain unchanged.